.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,33 +1,22 @@
 <!-- Provide a general summary of your changes in the Title above -->
 
+
 ## Description
-<!-- Describe your changes in detail -->
+<!-- Describe your changes in detail. -->
+<!-- If possible, please indicate if this change breaks or may break existing functionality, so we can mark it as 'breaking'. -->
+
 
 ## Motivation and Context
 <!-- Why is this change required? What problem does it solve? -->
 <!-- If it fixes an open issue, please link to the issue here. -->
 
-## Types of Changes
-<!-- Please select all items that apply either now or after creating the pull request: -->
-- [ ] Documentation update
-- [ ] Bug fix
-- [ ] New feature
-- [ ] Breaking change<sup>1</sup>
-
-<sup>1</sup>The change breaks (or has the potential to break) existing functionality.
 
 ## Checklist:
-<!-- Please select all items that apply either now or after creating the pull request. -->
+<!-- Please complete the checklist before the PR can be merged. Reviews can begin while the checklist is in progress. -->
```

Comment:
The part about reviews seems unnecessary. Reviews can begin at any point, even on draft PRs. Also made the language more direct/clear.
```suggestion
<!-- This checklist must be complete before merging the pull request. -->
```

Reply:
I was debating whether to keep the "please" :)